### ISSUE TITLE
Request/Response/Detail tab is initially scrolled to the bottom

### ISSUE THREAD

**Issue by Prinzhorn** [mitmproxy/mitmproxy#5089]
#### Problem Description
The response tab does not start at the top where the headers are but is scrolled all the way to the end.

#### Steps to reproduce the behavior:
1. `mitmproxy`
2. `curl --insecure --proxy localhost:8080 https://example.com`
3. Open the flow in the UI
4. Open the Response tab

![Selection_1096](https://user-images.githubusercontent.com/679144/150962219-890a511a-0070-44ee-a32a-76d68fb58000.png)

#### System Information

```
Mitmproxy: 8.0.0.dev (+210, commit 83b43b9)
Python:    3.8.10
OpenSSL:   OpenSSL 1.1.1m  14 Dec 2021
Platform:  Linux-5.13.0-27-generic-x86_64-with-glibc2.29
```

# Cause

git-bisect says 278611bf778f3e89672a7ac53129f2a836f82e4a https://github.com/mitmproxy/mitmproxy/pull/4845, so this didn't affect only WebSockets but all views?

**Comment by mhils** [comment]
Seems like that's the case. Thanks for catching this one.

**Comment by mhils** [comment]
I've reverted 278611b for the time being, which is the lesser of both evils. Of course the intention is to fix #4491 properly again.
